spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -90,7 +90,9 @@
         "spark",
         SparkInterpreter.class.getName(),
         new InterpreterPropertyBuilder()
-            .add("spark.app.name", "Zeppelin", "The name of spark application.")
+            .add("spark.app.name",
```

Comment:
ok.
I think the different mechanics facilitate different approaches to config management.
It will good if you could share and discuss with the community with which works well.